src/helpers/collateralsApy/index.js

```diff
@@ -55,8 +55,11 @@ export const fetchTokenApy = async (pool) => {
   }
 
   if (chainId === 42161) {
-    if (pool.config.id === 2 || pool.config.id === 3)
-      return await getGlpApy(pool.config.id === 3);
+    if (pool.config.id === 2 || pool.config.id === 3) {
+      const response = await getMegicGlpApy(chainId);
+      if (pool.config.id === 2) return response.glpApy;
```

Comment:
use be something like pool.config.collateralApyType instead and switch/case it

Reply:
This file is not related to the Stake refactoring. I'll fix it in the next updates